stdlib/nats/src/main/java/org/ballerinalang/nats/nativeimpl/producer/Send.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.nats.nativeimpl.producer;
+
+import io.nats.streaming.StreamingConnection;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.bre.bvm.CallableUnitCallback;
+import org.ballerinalang.connector.api.Struct;
+import org.ballerinalang.model.NativeCallableUnit;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.model.values.BValueArray;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.Receiver;
+import org.ballerinalang.nats.nativeimpl.Constants;
+import org.ballerinalang.nats.nativeimpl.Utils;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Sends message to a given subject.
+ *
+ * @since 0.995
```

Comment:
Shall we add `@since` to other classes as well to be consistent?

Reply:
This is added to all the classes which represents ballerina native function implementations.